modules/op-running-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -13,15 +13,7 @@ In the event of a pull request or a merge request, {pac} also runs pipelines fro
 * The author is the owner of the repository.
 * The author is a collaborator on the repository.
 * The author is a public member on the organization of the repository.
-* The pull request author is listed in an `OWNER` file located in the repository root of the `main` branch as defined in the GitHub configuration for the repository. Also, the  pull request author is added to either `approvers` or `reviewers` section. For example, if an author is listed in the `approvers` section, then a pull request raised by that author starts the pipeline run.
-
-[source,yaml]
-----
-...
-  approvers:
-    - approved
-...
-----
+* The pull request author is listed in the `approvers` or `reviewers` section of the `OWNERS` file in the root of the repository, as defined in the https://www.kubernetes.dev/docs/guide/owners/[Kubernetes documentation]. {pac} supports the full specification for the `OWNERS` and `OWNERS_ALIASES` files.
```

Comment:
I don't think we support the full specification but only a subset, @enarha would know more about which ones.

Reply:
Yea, the same sentence repeats few times. I commented on the first instance (https://github.com/openshift/openshift-docs/pull/75409#discussion_r1587186853), but it applies here as well.

Reply:
@mramendi the update above is good, just apply it here as well.